src/server/db/migrations/schema_undefined.md

```diff
@@ -0,0 +1,110 @@
+# Data Schema version: undefined
```

Comment:
Same as above, this doc belongs to the top "/docs" directory. If someone looks for what has been changed from one schema to another, it will be difficult to read and compare each individual schema documentation.

You probably could consider to have one data schema documentation that 1. describe all data schemas; 2. describe changes from one schema to the next. A method I tried before was to use diagram to highlight changes between schemas: https://github.com/fluid-project/preferencesServer/blob/main/doc/dataModel.md